### ISSUE TITLE
Fix component tests for Svelte v4

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#10682]
In https://github.com/webdriverio/webdriverio/commit/ef2af297466a258a76adc29f1fe3639b4f2f2e70 we have disabled Svelte component e2e tests as they fail with:

```
[plugin:vite:import-analysis] Missing "./internal/disclose-version" specifier in "svelte" package
```

which might be related to Svelte v4 and some incompatibility how we integrated it. See also https://github.com/sveltejs/svelte/issues/8869

**Comment by dummdidumm** [comment]
Which version (down to the patch version) are you using of
- Svelte
- Vite
- vite-plugin-svelte
- `@testing-library/svelte`

when this happens? Svelte 4.0.0 had `internal/disclose-version` as an `import` condition which was changed to `default` which fixed some related issue (but AFAIK Vite should handle both). Vite 4.4.0 had an issue that caused this exact error that was fixed in 4.4.1

**Comment by christian-bromann** [comment]
Thanks for the context. This will help resolve this.
